test/ui-test/testSuites/suite_status/tst_ChatFlow/test.feature

```diff
@@ -54,3 +54,9 @@ Feature: Status Desktop Chat
  #   Scenario: User cannot delete another user's message
  #        When user joins chat room test
  #        Then the user cannot delete the last message
+
+
+    Scenario: User can send a sticker after installing a free pack
+         When user joins chat room automation-test
+         Then the user can install the sticker pack with id 15
+         Then the user can send the sticker at index 2
```

Comment:
Just a minor comment here.. Reading it without context, I'm not sure what does it mean sending the sticker at `index 2`. It seems something about code not something that the user should be able to validate. Could we reformulate it to be more "user jargon"?